woocommerce-gateway-ebanx/templates/checkout-conversion-rate.php

```diff
@@ -13,7 +13,7 @@
 	<span style="float: left;"><?php echo $message; ?></span>
 	<img
 		class="ebanx-spinner" 
-		src="/wp-admin/images/spinner-2x.gif" 
+		src="<?= esc_url( admin_url('/images/spinner-2x.gif') ) ?>" 
```

Comment:
We are following a standard on all code, using `<?php echo ...` instead of `<?=`.